perfdash/parser.go

```diff
@@ -52,6 +52,30 @@ func createRequestCountData(data *perftype.DataItem) error {
 	return nil
 }
 
+type ContainerRestartsInfo struct {
+	Container    string
+	Pod          string
+	Namespace    string
+	RestartCount int
+}
+
+func parseContainerRestarts(data []byte, buildNumber int, testResult *BuildData) {
+	build := fmt.Sprintf("%d", buildNumber)
+	restarts := make([]ContainerRestartsInfo, 0)
+	if err := json.Unmarshal(data, &restarts); err != nil {
+		klog.Errorf("error parsing JSON in build %d: %v %s", buildNumber, err, string(data))
+		return
+	}
+
+	restartsbyContainer := make(map[string]float64)
```

Comment:
nit: s/restartsbyContainer/restartsByContainer

Reply:
Looks like both parts (restartsbyContainer and restartsbyContainer) are the same?

![image](https://user-images.githubusercontent.com/2559168/160351874-7dcddbac-8d53-4a41-a218-0f1d927f7bc4.png)

Reply:
There is capital **B**

Reply:
:D thanks!

Done